src/map/packets/entity_update.cpp

```diff
@@ -68,10 +68,7 @@ void CEntityUpdatePacket::updateWith(CBaseEntity* PEntity, ENTITYUPDATE type, ui
         case ENTITY_SPAWN:
         {
             updatemask = UPDATE_ALL_MOB;
-            if (PEntity->objtype == TYPE_PET)
-            {
-                ref<uint8>(0x28) = 0x04;
-            }
+
```

Comment:
So happy to see this go away. I looked at the history and this code goes back 11 years. Someone added it in not knowing what it actually was or needed to do and it has just stayed around that entire time!